simpleio.py

```diff
@@ -101,6 +102,67 @@ def shift_out(dataPin, clock, value, msb_first=True):
             tmpval = bool((value & (1 << i)))
             dataPin.value = tmpval
 
+class Servo:
```

Comment:
I changed servoMin/servoMax to 

> min_pulse 
and 
> max_pulse  

to conform with the Google Python Style Guide